tests/test_authority.py

```diff
@@ -115,25 +116,37 @@ def setUp(self):
             # Here the test is to confirm the OIDC endpoint contains no "/v2.0"
             self.authority + "/.well-known/openid-configuration")
 
+    def setup_tenant_discovery(self, tenant_discovery):
```

Comment:
Is acceptance test 2 from Ray's list covered?

Acceptance test case 2: MSAL shall error out
contoso.com/.well-known/openid-configuration returns {"issuer": "login.microsoftonline.com/tenant", ...}